src/utils/screengrabber.cpp

```diff
@@ -205,11 +205,30 @@ QPixmap ScreenGrabber::grabEntireDesktop(bool& ok)
 #endif
 #if defined(Q_OS_LINUX) || defined(Q_OS_UNIX) || defined(Q_OS_WIN)
     QRect geometry = desktopGeometry();
-    QPixmap p(QApplication::primaryScreen()->grabWindow(
-      wid, geometry.x(), geometry.y(), geometry.width(), geometry.height()));
-    QScreen* screen = qApp->screenAt(QCursor::pos());
-    p.setDevicePixelRatio(screen->devicePixelRatio());
-    return p;
+
```

Comment:
I'm good with this portion of the PR. It will only impact X11 and not wayland and seems to improve the X11 situation .